packages/mc-scripts/package.json

```diff
@@ -55,12 +55,12 @@
     "postcss-reporter": "6.0.1",
     "postcss-safe-parser": "4.0.1",
     "promise": "8.0.3",
-    "react-dev-utils": "7.0.3",
+    "react-dev-utils": "9.0.1",
```

Comment:
The breaking change here is the method signature of `createCompiler` to accept an object. I fixed that in `mc-scripts`.